modules/cloudant/src/test/java/com/ibm/cloud/cloudant/v1/model/SearchResultTest.java

```diff
@@ -27,6 +27,7 @@
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
```

Comment:
Odd this was added since it's not being used.

Reply:
Indeed, but if I see the other util classes are also not necessarily here except the HashMap.